testsuite/features/secondary/proxy_retail_pxeboot_and_mass_import.feature

```diff
@@ -28,8 +28,12 @@ Feature: PXE boot a Retail terminal
     And I manually install the "vsftpd" formula on the server
     And I manually install the "saltboot" formula on the server
     And I manually install the "pxe" formula on the server
+    And I manually install the "image-sync" formula on the server
```

Comment:
please remove the "pxe" and "image-sync" installations, they are not needed:
```
suma-bv-43-srv:~ # rpm -q patterns-suma_retail --requires 
bind-formula 
branch-network-formula 
dhcpd-formula 
image-sync-formula 
pxe-formula
```

Reply:
Ok, but my doubt is... where are we triggering the installation of that pattern in our CI proxy?

Reply:
Shouldn't we use the steps described here? https://documentation.suse.com/suma/4.3/en/suse-manager/retail/retail-install-setup.html#_configure_services_on_the_branch_server

So... installing the formulas using `retail_branch_init` ?

Reply:
> Ok, but my doubt is... where are we triggering the installation of that pattern in our CI proxy?

here https://github.com/uyuni-project/uyuni/blob/363de44c8385addc31bc3fbb399d440d9375a329/testsuite/features/core/proxy_branch_network.feature#L31

I am more and more convinced that the order of scenarios in

testsuite/features/core/proxy_branch_network.feature
testsuite/features/secondary/proxy_retail_pxeboot_and_mass_import.feature

is completely wrong and needs refactoring. It already looks much better in BV Retail.

Also, the second file needs being split again to separate base retail and mass import scenarios.

Reply:
> Shouldn't we use the steps described here? https://documentation.suse.com/suma/4.3/en/suse-manager/retail/retail-install-setup.html#_configure_services_on_the_branch_server
> 
> So... installing the formulas using `retail_branch_init` ?

I wrote those scenarios before that script got documented.

Reply:
It was a good reason to merge these two features. https://github.com/SUSE/spacewalk/pull/9452

Reply:
I think this complete one scenario can be removed if we already have pattern installed. But then OTOH, why did @nadvornik feel the need to add image-sync here?

If we indeed need to install anything here, then I suggest to install pattern as @Bischoff does. Installing pattern is the supported and documented way (either explicitly or implicitly by SUMA Retail product).

Reply:
yes, I have already tested that all formulas are installed via pattern and this scenario can be removed